climpred/bootstrap.py

```diff
@@ -23,26 +23,27 @@
     shift_cftime_singular,
 )
 
+concat_kwargs = {'coords': 'minimal', 'compat': 'override'}
```

Comment:
Make this all caps for constant. Can it go into `constants.py` or is there a circular imports problem?

Reply:
works. a nicer version would be a small function that checks whether this is applied to xr.dataset or xr.dataarray and then adds data_vars='minimal'

Reply:
I think that would be hard to implement cleanly. You'd probably have to have some `xr.concat` wrapper that you call wherever we concat and do that check. You can try it here or we can just leave it for another time. The current kwargs probably handle most of it.